packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -69,54 +88,48 @@ export const getPatientByDemo = async ({
   facilityId: string;
   cxId: string;
   demo: PatientData;
-}): Promise<Patient | null> => {
-  const { log } = Util.out(`getPatientByDemo - cxId ${cxId}`);
+}): Promise<Patient | undefined> => {
+  // Normalize the patient demographic data
+  const normalizedPatientDemo = normalizePatientDataMPI(convertPatientDataToPatientDataMPI(demo));
+  if (!normalizedPatientDemo) {
+    return undefined;
+  }
 
-  const patients = await PatientModel.findAll({
-    where: {
-      cxId,
-      facilityIds: {
-        [Op.contains]: [facilityId],
-      },
-      data: {
-        dob: demo.dob,
-        genderAtBirth: demo.genderAtBirth,
-      },
+  // Block patients based on the criteria of matching dob and genderAtBirth
+  const blockedPatients = await blockPatients({
+    cxId,
+    facilityIds: [facilityId],
```

Comment:
Why limiting to the patients of the same facility?

Reply:
good point, but also, if we are returning a patient from one facility back to the customer at a different facility, then I think we need to add the new facility to the facility list on the patient. right?

Reply:
good point, I agree we should add the facility ID - I'd just confirm this on slack as its not just technical but a product decision

Reply:
In the slack thread we decided that for now, doesnt make sense to add the facilityID back